File: RESOLUCOES/python/lista/01/ex012.py
# Crie um programa que leia o preço de um produto, calcule e mostre o seu PREÇO PROMOCIONAL, com 5% de desconto.

valor = float(input('Digite um preço para calcular seu desconto de 5%: '))

desconto = valor - valor * 0.05
print(f'O preço com desconto é de: R${desconto:.2f}')

File: resolucoes/java/lista01/src/main/java/org/example/Ex012.java

/*Crie um programa que leia o preço de um produto, calcule e mostre o seu 
PREÇO PROMOCIONAL, com 5% de desconto. */

package org.example;

import java.util.Scanner;

public class Ex012 {
    public static void main(String[] args) {
        Ex012.main();
    }

    public static void main() {

        Scanner sc = new Scanner(System.in);
        System.out.print("Digite o preço do produto: R$");
        Float preco = sc.nextFloat();
        System.out.printf("O preço do produto com 5%% de desconto é R$%s", preco - preco / 100 * 5);
        sc.close();

    }

}
